Zoo/src/test/java/BuildingManagerTest.java

```diff
@@ -0,0 +1,59 @@
+import org.junit.Test;
+import ua.lviv.iot.algo.part1.lab3.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+
+public class BuildingManagerTest extends BuildingManager {
+    List<Building> buildings = new ArrayList<>();
+
+    @Test
+    public void testFindBuildingsBuiltAfter() {
+        List<Building> buildings = new ArrayList<>();
+
+        Building building1 = new Zoo("Hoshin", "Tokyo", 200.5, 100, false, 2001);
+        Building building2 = new Bank(8, "10:00-21:00", 25, false, 1996);
+        Building building3 = new School(720, 50, "LPML", true, 1980);
+        Building building4 = new Library(10000, 5, "Stepana Bandery str.", false, 2004);
+
+        buildings.add(building1);
+        buildings.add(building2);
+        buildings.add(building3);
+        buildings.add(building4);
+
+        List<Building> expectedBuildings = new ArrayList<>();
+        expectedBuildings.add(building1);
+        expectedBuildings.add(building4);
+
+        List<Building> actualBuildings = buildings.stream().
+                filter(building -> building.getYearOfBuilding() > 2000).
+                collect(Collectors.toList());
```

Comment:
Use the actual method when testing. I suppose you tried to test findBuildingsBuiltAfter() method, but instead you write the functionality right in test